ReconstructOrder/workflow/multiDimProcess.py

```diff
@@ -329,23 +329,32 @@ def process_sample_imgs(img_io: Union[mManagerReader, PolAcquReader]=None,
                        'Retardance+Orientation', 'Polarization+Orientation', 'Brightfield+Retardance+Orientation',
                        'Brightfield_computed+Retardance+Orientation',
                        'Retardance+Fluorescence', 'Retardance+Fluorescence_all']
-    phase_names     = ['Phase2D', 'Phase_semi3D', 'Phase3D']
+    ret_Zavg_names  = ['RetardanceZavg', 'OrientationZavg']
+    phase_names     = ['Phase2D', 'Phase_semi3D', 'Phase3D', 'Absorption2D', 'Absorption_semi3D']
     fluor_names     = ['405', '488', '568', '640', 'ex561em700']
 
+
     # 2 )set flags based on names defined in 1)
     save_stokes    = any(chan in stokes_names + stokes_names_sm for chan in img_io.output_chans) \
-                     or save_stokes_fig
+                 or save_stokes_fig
     save_phase     = any(chan in phase_names for chan in img_io.output_chans)
-    save_birefring = any(chan in birefring_names for chan in img_io.output_chans) or save_fig or save_phase
+    save_ret_Zavg  = any(chan in img_io.output_chans for chan in ret_Zavg_names)
+    save_birefring = any(chan in birefring_names for chan in img_io.output_chans) or save_fig or save_phase or save_ret_Zavg
     save_BF        = 'Brightfield' in img_io.output_chans
     save_pol       = any(chan in pol_names for chan in img_io.output_chans) or save_pol_fig
     save_fluor     = any(chan in fluor_names for chan in img_io.output_chans)
 
+    have_pol_data = any([any([substring in chan_name for substring in ['State', 'state', 'Pol']])\
+                         for chan_name in img_io.input_chans])
+    if not have_pol_data:
+        save_birefring = False
```

Comment:
Should we maybe throw an warning or error message if there is no pol data but the user tries to reconstruct birefirefringence and/or phase?